candlepin.spec.tmpl

```diff
@@ -234,6 +234,9 @@ fi
 ## else logroate fails
 %attr(750, tomcat, tomcat) %{_localstatedir}/log/%{name}
 
+## logback.xml should not be overridden during rpm upgrade/install
+%config(noreplace) %{_sharedstatedir}/tomcat/webapps/%{name}/WEB_INF/classes/logback.xml
```

Comment:
I tried building the rpm with tito but it fails with `File not found: /tmp/tito/rpmbuild-candlepinvn3f8h5p/BUILDROOT/candlepin-4.2.7-1.x86_64/var/lib/tomcat/webapps/candlepin/WEB_INF/classes/logback.xml`
I think `WEB_INF` is supposed to be `WEB-INF`

Reply:
Aside from this, the change does the job: an rpm upgrade that includes changes to the logback.xml results in a new file being created without overriding the old one: `warning: /var/lib/tomcat/webapps/candlepin/WEB-INF/classes/logback.xml created as /var/lib/tomcat/webapps/candlepin/WEB-INF/classes/logback.xml.rpmnew`